ondewo/nlu/intent.proto

```diff
@@ -127,17 +127,17 @@ service Intents {
         };
     }
 
-    // Creates a training phrase for a specified intent (of a specific intent).
-    rpc CreateTrainingPhrase(CreateTrainingPhraseRequest) returns (Intent.TrainingPhrase);
+    // Creates batch of training phrases
+    rpc CreateTrainingPhraseBatch(CreateTrainingPhraseBatchRequest) returns (TrainingPhraseStatusBatch);
 
-    // Retrieve a specific training phrase (of a specific intent).
-    rpc GetTrainingPhrase(GetTrainingPhraseRequest) returns (Intent.TrainingPhrase);
+    // Retrieve a specific training phrases batch (of a specific intent).
```

Comment:
it is not of a single "specific intent".

Same for DeleteBatch.

They all are independent from the intent since they have IDs.
In the case of CreateBatch, you need to pass the intent_name per phrase also.

Reply:
have changed the comments